client/basic-authorship/src/basic_authorship.rs

```diff
@@ -132,7 +132,7 @@ impl<A, B, Block, C> sp_consensus::Environment<Block> for
 
 /// The proposer logic.
 pub struct Proposer<B, Block: BlockT, C, A: TransactionPool> {
-	inner: Arc<ProposerInner<B, Block, C, A>>,
+	inner: ProposerInner<B, Block, C, A>,
```

Comment:
We could remove `ProposerInner` now and put its fields here right?

Reply:
Good point